src/CommentFlagProcessor.ts

```diff
@@ -51,11 +51,12 @@ export class CommentFlagProcessor {
         let affectedRange: Range;
         if (tokenized.disableType === 'line') {
             affectedRange = util.createRange(range.start.line, 0, range.start.line, range.start.character);
-        } else if (tokenized.disableType === 'next-line') {
+        } else {
+            // tokenized.disableType must be 'next-line'
```

Comment:
The type of `tokenized.disableType` is `'line' | 'next-line'`. If this type is wrong and some third value is possible, then it would be best to not remove the `if` here, and instead either:

- Add a check for the invalid value and throw an error
- Short-circuit and return

I think that the chances of an invalid third value getting in here are low, so I removed the conditional